package.json

```diff
@@ -36,7 +36,9 @@
     "node-sass": "^4.12.0",
     "prop-types": "^15.7.2",
     "react": "^16.10.2",
+    "react-cookies": "^0.1.1",
     "react-dom": "^16.10.2",
+    "react-router": "^5.1.2",
```

Comment:
what is this for "react-router"

Reply:
the storybook:build return error without this library, but I am not actually using it

Reply:
that is why you should explain it in PR, otherwise i will ask you~